R/maic_anchored.R

```diff
@@ -154,9 +154,9 @@ maic_anchored <- function(weights_object,
 
   # : assign weights to real and pseudo ipd
   if (normalize_weights) {
-    ipd$weights <- weights_object$data$scaled_weights[match(weights_object$data$USUBJID, ipd$USUBJID)]
+    ipd$weights <- weights_object$data$scaled_weights[match(ipd$USUBJID, weights_object$data$USUBJID)]
```

Comment:
@MikeJSeo Can you add a test that fails on the original version and is fixed with this change?